src/style.css

```
@@ -1 +1,67 @@
 /* Styles go here */
+/* * {
+  margin: 0;
+  box-sizing: border-box;
+} */
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  text-transform: uppercase;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.nav_item {
+  margin-left: 20px;
+}
+
+.link {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.link.is-active::after {
```

don't increase selectors specificity
```suggestion
.is-active::after {
```